Scripts/Sources/Automation/Eval.swift

```diff
@@ -0,0 +1,329 @@
+import Foundation
+import PathKit
+import xcproj
+
+class Eval {
+    static func main() {
+        print("💁🏻‍♂️ Job type: \(TravisCI.jobType().description)")
+        
+        if isSpecificJob() {
+            return
+        }
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tevelee/Eval/blob/0e45c7188f4e60e9487408f3d9b69e8a8f17f368/Scripts/Sources/Automation/Eval.swift#L12"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>